### ISSUE TITLE
last names starting with Mc, Van, O' etc have a space between the first and second part of the name

### ISSUE THREAD

**Issue by EveSnyder** [historyforge/historyforge-old#187]
I've noticed this in our HF and now Auburn and Oberlin too.

**Comment by dfurber** [comment]
Interesting...

**Comment by dfurber** [comment]
The cause is using the `titleize` method to correct capitalization had the unwanted side effect of adding that space. The fix is to make it find a different way to ensure capitalization that respects name particles.

**Comment by dfurber** [comment]
And the names should also be reverted back to the correct names.

**Comment by EveSnyder** [comment]
It did revert in our HF but not Oberlin, I can't check Auburn from computer but I'm guessing its the same.

**Comment by dfurber** [comment]
That's because it can't revert something that started that way. Those will have to be corrected by hand unfortunately.

**Comment by EveSnyder** [comment]
Okay I'll let them know.  Also I just noticed on ours that some of the names which started with Mac were fixed and others weren't.  One was from October-I didn't realize the problem went back that far.

**Comment by dfurber** [comment]
Here's the deal. I found a proper name capitalization module. I wrote a script that ran through all the census records. If the previous capitalization scheme had changed the name, then it took the name before it was changed and ran the new name capitalization on it. Otherwise it left it. So it is possible that it was entered wrong but in a way that the old capitalization scheme didn't change it, so it got ignored by the script. Saving the record again would fix it. Or I could make it save all the records again and that would apply the new capitalization scheme. Ack!
